### ISSUE TITLE
py.test not found

### ISSUE THREAD

**Issue by memeplex** [pghilardi/atom-python-test#26]
I'm getting this

```
Running tests... 
 
An error occured while executing py.test. Check if py.test is installed and is in your path.
```

despite having the py.test command installed under `~/.local/bin` which is in my PATH. I'm able to run py.test normally from a terminal.

**Comment by memeplex** [comment]
I see the problem is that atom-python-test is calling `python -m pytest` instead of simply executing `py.test` and I'm using `python3`. Is it possible to add a configuration option to set the python executable? I'm also using other packages like hydrogen and python-ide that don't seem to have this problem, somehow they detect the correct python version. I know I can set a virtualenv but it would be nice to get this running in my barebones user environment.

**Comment by ghost** [comment]
Any solutions here?

**Comment by MaciejStrzempek** [comment]
Hi everyone. I was struggling with the same problem. As memeplex mention atom-python-test is calling `python -m pytest` command. I was searching through files and I was able to found where the command is. In your `~/.atom/packages/atom-python-test/lib` there is file called atom-python-test.coffee if you change line 93 form  `command = 'python'` to  `command = 'python3'` everything should work fine. I agree that there is a lack of ability to do this from setting menu but for now, this not so elegant solution should work.

**Comment by pghilardi** [comment]
Sorry for the late reply guys. I have fixed this on version 0.10.0, adding the option to customize the python executable path, so you can configure to python3 executable.

Thanks!

**Comment by github-sjacobs** [comment]
I am seeing this same error. I am a bit confused about how this is supposed to work. I have atom-python-virtualenv installed (as suggested) and it seems to work fine. I can select a virtual environment and open a terminal and use that environment.

The solution above may well work but shouldn't the path to the python executable come from the virtual environment? If I set the path in the settings I assume it will not necessarily use the current virtual environment.
